xmonad.cabal

```diff
@@ -71,7 +71,7 @@ library
                    setlocale,
                    mtl,
                    process,
-                   unix,
+                   unix >= 2.8.0.0,
```

Comment:
I don't think that depending on the latest version is a viable solution. `unix` is a core library of `ghc` and as such cannot be updated easily.